core/__init__.py

```diff
@@ -252,7 +252,21 @@ def setup(modules: Union[object, ModuleType], render: Union[ModuleType, Dict] =
             assert uri is not None and (uri.lower().startswith("https://") or uri.lower().startswith("http://"))
     runStartupTasks()  # Add a deferred call to run all queued startup tasks
     initializeTranslations()
-    assert conf["viur.file.hmacKey"], "You must set a secret and unique Application-Key to viur.file.hmacKey"
+    if conf["viur.file.hmacKey"] is None:
+        from viur.core import db
+        key = db.Key("viur-conf", "viur-conf-modulekey")
```

Comment:
Can you please rename this to just kind "viur-conf", key "viur-conf"`
```suggestion
        key = db.Key("viur-conf", "viur-conf")
```